src/main/java/dev/slimevr/protocol/DataFeedBuilder.java

```diff
@@ -176,8 +163,10 @@ public static int createDeviceData(FlatBufferBuilder fbb, int id, DeviceDataMask
 		return DeviceData.endDeviceData(fbb);
 	}
 
-	public static int createSyntheticTrackersData(FlatBufferBuilder fbb, TrackerDataMaskT trackerDataMaskT, List<Tracker> trackers) {
-		if (trackerDataMaskT == null) return 0;
+	public static int createSyntheticTrackersData(FlatBufferBuilder fbb, TrackerDataMaskT trackerDataMaskT,
```

Comment:
Same thing as my prior comment about newline behavior on function chaining and arguments. In this case, I would prefer that when a wrap is necessary, each parameter gets its own newline.